### PR TITLE
ZEN-15235: "Sort Column" and "Sort Sense" don't work on Custom Device Re...

### PR DESCRIPTION
**Pull request by serge30**
...port

Port: https://jira.zenoss.com/browse/ZEN-15235

### CODE REVIEW

File: Products/ZenWidgets/ZenTableManager.py [hunk 1]
```diff
@@ -143,6 +143,15 @@ def setReqTableState(self, tableName, attrname, default=None, reset=False):
                                         default=default, reset=reset)
 
 
+    def deleteTableState(self, tableName):
+        """delete an existing table state"""
+        request = self.REQUEST
```
Comment (jpeacock-zenoss) [inline Products/ZenWidgets/ZenTableManager.py]: Is there a reason for this line of code?

Comment (serge30) [inline Products/ZenWidgets/ZenTableManager.py]: ups... no reasons, removed.

**Comment by jpeacock-zenoss** [comment]
Why was the `zmanage_editProperties` (and `deleteTableState`) method added.  How does that pertain to the solution?

**Comment by serge30** [comment]
Sorting parameters (field and direction) are stored in 2 places. One is the Custom Report properties which are edited on UI and stored in ZODB. The second one is table state which holds last column used for sorting. The table state has a priority over Custom Report properties. 

So after editing the report properties, new sorting parameters were not applied and this method resets table state.

**Pull request merged**